descriptors/ModuleDescriptor-template.json

```diff
@@ -276,7 +280,9 @@
           "methods": [ "GET" ],
           "pathPattern": "/bulk-edit/permissions-self-check",
           "permissionsRequired": ["bulk-edit.permissions-self-check.get"],
-          "modulePermissions": ["perms.users.get"]
+          "modulePermissions": [
+            "perms.users.get",
+            "permissions.users.item.get"]
```

Comment:
Please, move brakes to the next line to follow common format of module descriptor.

Reply:
Updated